components/emoji_picker/emoji_picker_tabs.jsx

```diff
@@ -11,7 +11,7 @@ import {makeAsyncComponent} from 'components/async_load';
 import EmojiPicker from 'components/emoji_picker';
 import EmojiPickerHeader from 'components/emoji_picker/components/emoji_picker_header';
 
-const GifPicker = makeAsyncComponent('GifPicker', React.lazy(() => import('components/gif_picker/gif_picker.jsx')));
+const GifPicker = makeAsyncComponent('GifPicker', React.lazy(() => import('components/gif_picker/gif_picker')));
```

Comment:
Not for React components like this. They have to be capitalized when rendered, otherwise React will output something like `<gifpicker/>` as raw HTML because it assumes that `gifpicker` is an HTML element

Reply:
yes the convention is to use snake case for file names across webapp. That is also how the webpack is compiled to help with lazy imports

Reply:
Oh, I see. We should be using snake case everywhere like Nevy mentioned. These files don't follow that because they were actually implemented by someone from Gfycat, and I seem to remember there was some argument over having them follow our coding and naming styles that we ended up settling on. It's all pretty self-contained, so that's why I haven't pushed to have them changed, and I don't know if anyone else cares enough to do anything about that either. 😛 

To answer your original question though, the differences in naming style don't have any meaning for lazy imports or anything like that though

Reply:
Sorry for my late reply. Thanks to Nevyana and Harrison for the answers